source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_context.py

```diff
@@ -408,7 +410,12 @@ def step(self, render: bool = True):
             self.app.update()
 
         # step the simulation
-        super().step(render=render)
+        super().step(render=False)
+
+        self._step_count += 1
+
+        if self._step_count % self.cfg.render_interval == 0 and (self.has_gui() or self.has_rtx_sensors()):
+            self.render()
```

Comment:
To prevent crashing because of overflow.

```suggestion
            self.render()
            self._step_count = 0
```